docs/source/applications/image_denoising.rst

```diff
@@ -1,8 +1,143 @@
-Image Denoising
-===============
+.. code::
 
+    %%capture
+    !pip install kornia
+    !pip install kornia-rs
 
+.. code::
 
-.. raw:: html
+    import io
 
-    <gradio-app space="kornia/total_variation_denoising"></gradio-app>
+    import requests
+
+
+    def download_image(url: str, filename: str = "") -> str:
+        filename = url.split("/")[-1] if len(filename) == 0 else filename
+        # Download
+        bytesio = io.BytesIO(requests.get(url).content)
+        # Save file
+        with open(filename, "wb") as outfile:
+            outfile.write(bytesio.getbuffer())
+
+        return filename
+
+
+    url = "https://github.com/kornia/data/raw/main/doraemon.png"
+    download_image(url)
+
+
+
+
+.. parsed-literal::
+
+    'doraemon.png'
```

Comment:
all the code before this line, didn't matter for the docs itself, this makes more sense for the tutorial -- which you can include as a link into this page